test/remote/gateways/remote_cecabank_rest_json_test.rb

```diff
@@ -144,6 +144,39 @@ def test_failure_stored_credential_invalid_cit_transaction_id
     assert_match '810', purchase.error_code
   end
 
+  def test_purchase_without_exemption_type
```

Comment:
💬 Comment:

Is not bad per se, but here you are doing a remote test just to check the request body, not the actual response, this could perfectly be a unit test.

Reply:
Good point, I will remove it